build/deploy/examples/minimum.jsonnet

```diff
@@ -3,19 +3,20 @@ local dss = import '../../deploy/dss.libsonnet';
 local metadataBase = import '../../deploy/metadata_base.libsonnet';
 
 local metadata = metadataBase {
-  namespace: 'dss-main',
+  namespace: 'your_namespace', //Same namespace used in the make-certs.py and apply-certs.sh scripts
```

Comment:
nit: Let's cue the user into where to find this value just a little more with
`//Same <NAMESPACE> used in the make-certs.py and apply-certs.sh scripts`